pkg/kiln/io.go

```diff
@@ -295,3 +300,22 @@ func GetExampleDockerfile(dockerInfo *DockerInfo) (buf *bytes.Buffer, err error)
 
 	return buf, nil
 }
+
+// DetermineBaseImage consumes a given runtime selection and determines the base image for the Dockerfile
+func DetermineBaseImage(runtimeSelection string) (baseImage string, err error) {
+	runtimeSplit := strings.Split(runtimeSelection, ":")
+
+	switch runtimeSplit[0] {
```

Comment:
I get you.  I wouldn't map each `{runtime}:{version}`, just each `{runtime}` to a base image.